docs/user-guide/ze-profiles.md

```diff
@@ -42,94 +153,37 @@ Follow these steps:
-6. Create a profile.
-
-Your Zowe Explorer credentials are now stored securely.
-
-### For Zowe CLI users
-
-Ensure that you install the SCS plug-in for Zowe CLI before activating SCS in Zowe Explorer. For more information about the SCS plug-in for Zowe CLI, see [Secure Credential Store plug-in for Zowe Explorer](cli-scsplugin.md).
+1. Open Zowe CLI and run the following command:
 
-**Important:** If you did not install the SCS plug-in for Zowe CLI and try to activate SCS in the extension, you will not be able to use your existing profiles, and will have to recreate them.
-
-Activate the SCS plug-in in Zowe Explorer.
-
-1. Open Zowe CLI and issue the following command:
-
-   ```shell
-   zowe scs u
+   ```
+   zowe auth login apiml
    ```
 
-2. Open Zowe Explorer.
-3. Navigate to the VSCode settings.
-4. Open Zowe Explorer Settings.
-5. Add the **Zowe-Plugin** value to the `Zowe Security: Credential Key` entry field.
-6. Restart VSCode.
-
-The credentials of your newly created or existing profiles are now stored securely.
-
-## Use Base Profile and Token with Existing Profiles
-
-As a Zowe user, you can leverage the base profile functionality to access multiple services through Single Sign-on. Base profiles enable you to authenticate via Zowe API Mediation Layer. You can use base profiles with more than one service profile. For more information, see [Base Profiles](https://docs.zowe.org/stable/user-guide/cli-usingcli#base-profiles).
-
-Before you log in and connect your service profile, ensure that you have [Zowe CLI](https://docs.zowe.org/stable/user-guide/cli-installcli.html) v6.16 or higher installed.
-
-### Access services through API ML with SSO
-
-Connect your service profile with a base profile and token.
-
-**Follow these steps:**
-
-1. Open Zowe CLI and run the following command: `zowe auth login apiml`.
-
-2. Follow the instructions to complete the login process.
+2. Follow the on screen instructions to complete the login process.
 
-   A local base profile is created that contains your token. For more information about the process, see [Token Management](https://docs.zowe.org/stable/user-guide/cli-usingcli#how-token-management-works).
+   A local base profile is created that contains your token. For more information about the process, see [Token Management](../user-guide/cli-using-integrating-apiml.md#how-token-management-works).
   
 3. Run Zowe Explorer and click the **+** icon.  
 
 4. Select the profile you use with your base profile with the token.
 
    The profile appears in the tree and you can now use this profile to access z/OSMF via the API Mediation Layer.
 
-For more information, see [Integrating with API Mediation Layer](https://docs.zowe.org/stable/user-guide/cli-usingcli#integrating-with-api-mediation-layer).
+For more information, see [Integrating with API Mediation Layer](../user-guide/cli-using-integrating-apiml.md).
 
-### Log in to the Authentication Service
+### Logging in to the Authentication Service
```

Comment:
we should be careful using `.yaml` references for content that applies to both v1 and v2 profiles.
`The token is stored in the default base profile .yaml file.` should probably just say `The token is stored in the corresponding base profile.`